File: Rubik.HTML/Server/Schema/HierarchySet.cs
using System;
using System.Web;
using System.ComponentModel;
using System.Collections.Generic;

namespace Rubik.Server.Schema
{
    public class HierarchySet
    {
        public string HierarchyName
        {
            get; set;
        }

        public string LevelName
        {
            get; set;
        }

        public List<SelectedMember> SelectedMembers
        {
            get; set;
        }

        public List<string> Properties
        {
            get; set;
        }
    }
}

File: Rubik.HTML/Lib/Schema/HierarchySet.ts
/// <reference path="Interfaces.ts" />
/// <reference path="SelectedMember.ts" />

module Rubik.Server.Schema {
    export class HierarchySet implements IHierarchySet {
        public HierarchyName: string;
        public LevelName: string;
        public SelectedMembers: List<SelectedMember>;
        public Properties: List<string>;
    }
}